App.tsx

```diff
@@ -6,93 +6,26 @@
  */
 
 import React from 'react';
-import type {PropsWithChildren} from 'react';
-import {
-  SafeAreaView,
-  ScrollView,
-  StatusBar,
-  StyleSheet,
-  Text,
-  useColorScheme,
-  View,
-} from 'react-native';
-
-import {
-  Colors,
-} from 'react-native/Libraries/NewAppScreen';
-
-import Main from './src/pages/Main';
-
-type SectionProps = PropsWithChildren<{
-  title: string;
-}>;
-
-function Section({children, title}: SectionProps): JSX.Element {
-  const isDarkMode = useColorScheme() === 'dark';
-  return (
-    <View style={styles.sectionContainer}>
-      <Text
-        style={[
-          styles.sectionTitle,
-          {
-            color: isDarkMode ? Colors.white : Colors.black,
-          },
-        ]}>
-        {title}
-      </Text>
-      <Text
-        style={[
-          styles.sectionDescription,
-          {
-            color: isDarkMode ? Colors.light : Colors.dark,
-          },
-        ]}>
-        {children}
-      </Text>
-    </View>
-  );
-}
+import {ThemeProvider} from 'styled-components/native';
+import {SafeAreaView, StatusBar} from 'react-native';
+import {ContentView} from './styles';
+import {lightTheme, darkTheme} from './theme';
+import Main from './src/pages/Main/index';
+import {useColorScheme} from 'react-native';
 
 function App(): JSX.Element {
-  const isDarkMode = useColorScheme() === 'dark';
-
-  const backgroundStyle = {
-    backgroundColor: isDarkMode ? Colors.darker : Colors.lighter,
-  };
+  const isDarkTheme = useColorScheme() === 'dark'
 
   return (
-    <SafeAreaView style={backgroundStyle}>
-      <StatusBar
-        barStyle={isDarkMode ? 'light-content' : 'dark-content'}
-        backgroundColor={backgroundStyle.backgroundColor}
-      />
-    <View style={styles.body}>
-      <Main></Main>
-    </View>
-    </SafeAreaView>
+    <ThemeProvider theme={isDarkTheme? darkTheme : lightTheme }>
```

Comment:
mmm in this case you should use you custom theme, I put an example in the firts comment, 
if you want to implement dark and light themes it should be like this:

const theme = {
    light: {
       colors: {
       white: "#ffffff"
       }
    },
    dark: {
       colors: {
          white: "grey",
          .....
       }
    }
}